python/ray/util/spark/cluster_init.py

```diff
@@ -1213,19 +1249,23 @@ def setup_ray_cluster(
     cluster.
 
     Args:
-        num_worker_nodes: This argument represents how many ray worker nodes to start
+        max_worker_nodes: This argument represents maximum ray worker nodes to start
             for the ray cluster.
-            If autoscale=True, then the ray cluster starts with zero worker node,
-            and it can scale up to at most `num_worker_nodes` worker nodes.
+            If autoscale=True, then the ray cluster starts with "min_worker_nodes"
+            number of worker node, and it can scale up to at most `num_worker_nodes`
+            worker nodes.
+            If autoscale=False, then the ray cluster starts with "max_worker_nodes"
+            number of worker nodes.
```

Comment:
Now I'm thinking whether we still need the `autoscale` bool: if min_worker_nodes and max_worker_nodes are different then it implies that it's an autoscaling cluster. Otherwise it's a static cluster. The default value of `min_worker_nodes` can be the same as `max_worker_nodes` so by default, it's a static cluster.

Reply:
But we have released Ray 2.8 that contains "autoscale" argument. lets keep it. :) we also have 2 other arguments starting with "autoscal_" prefix. "autoscale=True" means we enable autoscaling explicitly

Reply:
We can make a one time breaking change when we GA ray-on-spark.